web/src/main/java/model/Account.java

```diff
@@ -0,0 +1,56 @@
+package model;
+
+
+import jakarta.persistence.*;
```

Comment:
Avoid using star imports. This is bad, because we don't have control over what we import from the package. Instead, modify your IntelliJ settings to not usse star import https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports